templates/components/multiagent/typescript/workflow/stream.ts

```diff
@@ -1,65 +1,71 @@
-import { StopEvent } from "@llamaindex/core/workflow";
+import { StopEvent, WorkflowContext } from "@llamaindex/workflow";
 import {
-  createCallbacksTransformer,
   createStreamDataTransformer,
   StreamData,
   trimStartOfStreamHelper,
-  type AIStreamCallbacksAndOptions,
 } from "ai";
-import { ChatResponseChunk } from "llamaindex";
+import { ChatResponseChunk, MessageContent } from "llamaindex";
 import { AgentRunEvent } from "./type";
 
-export function toDataStream(
-  result: Promise<StopEvent<AsyncGenerator<ChatResponseChunk>>>,
-  callbacks?: AIStreamCallbacksAndOptions,
-) {
-  return toReadableStream(result)
-    .pipeThrough(createCallbacksTransformer(callbacks))
-    .pipeThrough(createStreamDataTransformer());
-}
-
-function toReadableStream(
-  result: Promise<StopEvent<AsyncGenerator<ChatResponseChunk>>>,
-) {
+export async function createStreamFromWorkflowContext(
```

Comment:
@marcusschiesser here is the new stream handler code for the new workflow. The result is now included in the StopEvent from the context, so i added this function to handle for both text and data stream.

Reply:
Have to review that later

Reply:
btw, i think it's able to simplify this by transforming the events to the text stream as we did it in Python, but it looks to me that the Vercel SDK still prefer to use StreamData object but pipe it to the stream text instead, so i just let the two streams here. The best is we have a workflow/chatEngine.streamText() method in the framework.